### PR TITLE
dns.py docstrings, RO props

### PR DESCRIPTION
**Pull request by therealsilence**
- Format tweaks for readability
- Revise / add docstrings to forms and props
- Add read only ( 'ro' ) to properties that should not be modified once created

### CODE REVIEW

File: synapse/models/dns.py [hunk 1]
```diff
@@ -6,68 +6,87 @@ class DnsMod(CoreModule):
...
-                                'ex': 'vertex.link/1.2.3.4', 'doc': 'The result of a DNS A record lookup'}),
-                ('inet:dns:ns', {'subof': 'sepr', 'sep': '/', 'fields': 'zone,inet:fqdn|ns,inet:fqdn',
-                                 'ex': 'vertex.link/ns.dnshost.com', 'doc': 'The result of a DNS NS record lookup'}),
-                ('inet:dns:rev', {'subof': 'sepr', 'sep': '/', 'fields': 'ipv4,inet:ipv4|fqdn,inet:fqdn',
-                                  'ex': '1.2.3.4/vertex.link',
-                                  'doc': 'The transformed result of a DNS PTR record lookup'}),
-                ('inet:dns:aaaa', {'subof': 'sepr', 'sep': '/', 'fields': 'fqdn,inet:fqdn|ipv6,inet:ipv6',
-                                   'ex': 'vertex.link/2607:f8b0:4004:809::200e',
-                                   'doc': 'The result of a DNS AAAA record lookup'}),
+                ('inet:dns:look', {
```
Comment (invisig0th) [inline synapse/models/dns.py]: digging the readability breakdowns

File: synapse/models/dns.py [hunk 2]
```diff
@@ -6,68 +6,103 @@ class DnsMod(CoreModule):
...
+                    ('a:fqdn', {'ptype': 'inet:fqdn', 'doc': 'The domain queried for its DNS A record', 'ro': 1}),
+                    ('a:ipv4', {'ptype': 'inet:ipv4', 'doc': ' The IPv4 address returned in the DNS A record',
+                                'ro': 1}),
 
-                    ('ns', {'ptype': 'inet:dns:ns'}),
-                    ('ns:ns', {'ptype': 'inet:fqdn', 'ro': 1}),
-                    ('ns:zone', {'ptype': 'inet:fqdn', 'ro': 1}),
+                    ('ns', {'ptype': 'inet:dns:ns', 'doc': 'The DNS NS record returned by the lookup', 'ro': 1}),
+                    ('ns:zone', {'ptype': 'inet:fqdn', 'doc': 'The domain queried for its DNS NS record', 'ro': 1}),
+                    ('ns:ns', {'ptype': 'inet:fqdn', 'doc': 'The domain returned in the DNS NS record', 'ro': 1}),
```
Comment (vertexmc) [inline synapse/models/dns.py]: @therealsilence - should this be name server?

File: synapse/models/dns.py [hunk 3]
```diff
@@ -6,68 +6,103 @@ class DnsMod(CoreModule):
...
+                    ('a', {'ptype': 'inet:dns:a', 'doc': 'The DNS A record returned by the lookup', 'ro': 1}),
+                    ('a:fqdn', {'ptype': 'inet:fqdn', 'doc': 'The domain queried for its DNS A record', 'ro': 1}),
+                    ('a:ipv4', {'ptype': 'inet:ipv4', 'doc': ' The IPv4 address returned in the DNS A record',
+                                'ro': 1}),
 
-                    ('ns', {'ptype': 'inet:dns:ns'}),
-                    ('ns:ns', {'ptype': 'inet:fqdn', 'ro': 1}),
-                    ('ns:zone', {'ptype': 'inet:fqdn', 'ro': 1}),
+                    ('ns', {'ptype': 'inet:dns:ns', 'doc': 'The DNS NS record returned by the lookup', 'ro': 1}),
+                    ('ns:zone', {'ptype': 'inet:fqdn', 'doc': 'The domain queried for its DNS NS record', 'ro': 1}),
```
Comment (vertexmc) [inline synapse/models/dns.py]: @therealsilence - should this be name server?

File: synapse/cores/common.py [hunk 4]
```diff
@@ -1799,6 +1800,10 @@ def addTufoEvents(self, form, propss):
                     xact.fire('node:add', form=form, valu=valu, node=node)
                     xact.spliced('node:add', form=form, valu=valu, props=props)
 
+                if self.autoadd:
```
Comment (vertexmc) [inline synapse/cores/common.py]: @vEpiphyte - this "ro props" change looks good to me. If you want, we can merge this now as a separate PR.

**Pull request merged**